theme.json

```diff
@@ -66,10 +96,17 @@
         "appearanceTools": true,
         "useRootPaddingAwareAlignments": true,
         "layout": {
-            "contentSize": "1440px",
-            "wideSize": "1440px"
+            "contentSize": "var(--dswp-content-size)",
+            "wideSize": "var(--dswp-content-size)"
         },
         "typography": {
+            "fontFamilies": [
+                {
+                    "fontFamily": "BCSans",
```

Comment:
```suggestion
                    "fontFamily": "var(--dswp-typography-font-families-bc-sans), Noto Sans, Verdana, Arial, sans-serif",
```

Reply:
but then this gives people more options to choose??  This section is only for adding fonts to choose